api/src/main/resources/fireCloud.yaml

```diff
@@ -791,6 +791,15 @@ definitions:
       billingAccount:
         type: string
         description: the id of the billing account to use, must start with 'billingAccounts/'
+      highSecurityNetwork:
+        type: boolean
+        description: Optional, false if not specified. If true, spin up all compute in a VPC network.
+      enableFlowLogs:
+        type: boolean
+        description: Optional, false if not specified. If true, enable flow logs within the high security network. Requires highSecurityNetwork = true.
+      servicePerimeter:
```

Comment:
This has probably already been discussed elsewhere, but out of curiosity: do you have a plan for how this value will be provided? Single value per environment? Is there another API to create the perimeter, or is that a manual process?

Reply:
Each AoU env will have a named perimeter (https://docs.google.com/document/d/1r2bRy8CRul5JUJ5k9PjhyEfy9zx-cmXXiNmAemRMixQ/edit#heading=h.8695vxj8hxsq). Perimeter setup is semi-automated, I think CA team wrote some scripts to document / streamline the process, but we can bucket it under "one-time Terra environment setup" for now. We decided against doing a full-blown API for perimeter management, since it felt like overkill (at least for now).

Doing the servicePerimeter work will be the next step here, gated on getting a synthetic CDR housed in test.firecloud.org.